bftengine/tests/simpleTest/scripts/simpleTest.py

```diff
@@ -0,0 +1,465 @@
+                          int(cli.num_of_iterations), env,
+                          name.replace("+", "_"), r, customize_fn)
+
+
+def process_output(process_outputs):
+    res = ""
+    exit_code = 0
+    for output in process_outputs:
+        res += "{0} is done with exit code {1} and message '{2}'".format(
+            output.exec_name,
+            output.exit_code,
+            output.user_msg)
+        res += os.linesep
+        exit_code += output.exit_code
+    g_logger.debug(f"output: {res}, {exit_code}")
+    return TestResult(exit_code, res)
+
+
+def customize_test_vc_never_ends(test_config):
+    for o in test_config.process_outputs:
+        o.exit_code = 0
+        o.user_message = "Success"
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Simple test automation script",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("-bd",
+                        help="Folder with BFT executables",
+                        default="..")
+    parser.add_argument("-ld",
+                        help="Directory for log files",
+                        default=".")
+    parser.add_argument("-nc",
+                        help="Do not print processes' output to the console",
+                        action="store_true")
+    parser.add_argument("-l",
+                        help="Set log level for this script (not for the " +
+                        "executables)",
+                        default="INFO")
+    parser.add_argument("-i",
+                        help="Set number of operations to run",
+                        default="2800")
+    parser.add_argument("-vct",
+                        default=60000,
+                        type=int,
+                        help="View change timeout (if VC is enabled)")
+    parser.add_argument("-bft",
+                       default=["n=4,r=4,f=1,c=0,cl=1"],
+                       nargs="+",
+                       help="List of BFT configuration sets to use. All " +
+                       "parameters should be separated by ','. " +
+                       "n - total number of " + "replicas, r - number of " +
+                       "running replicas, " +
+                       "f - max. number of byzantine replicas, " +
+                       "c - max. number of slow replicas, " +
+                       "cl - number of running clients, " +
+                       "testViewChange - if present, viewChange will be " +
+                       "triggered after 1/2 of iterations done. " +
+                       "Please note, if View change is enabled AND n - r " +
+                       "<= f - the system will not progress after primary " +
+                       "is killed, and the test will SUCCESS after timeout " +
+                       "- this is desired behaviour for this configuration." +
+                       "You can provide set " +
+                       "set of configurations in the form of " +
```

Comment:
nit: The word "set" is repeated here.

Furthermore, I think it would read a little better as "**a** set" rather than "set".